src/ff/galois_field.rs

```diff
@@ -19,6 +19,7 @@ pub trait GaloisField:
 }
 
 // Bit store type definitions
+type U8_0 = BitArr!(for 1, in u8, Lsb0);
```

Comment:
This doesn't work as intended. I created a new PR to fix the issue.

tl;dr, You can specify `U8_1` for `Gf2`.

Longer version:
`U8_0` and `U8_1` generates the same struct which is `[u8; 1]`. We need to implement the `Block` trait to let the macro know of bit-lengths and sizes of the internal storage types, but the compiler will complain if we implement one for `U8_0` because we already have the conflicting implementation for `U8_1`.

Reply:
Is this resolved now?

Reply:
> Is this resolved now?
Yes, it's in the main. Please remove `U8_0` and just use the existing `U8_1`.